addon/components/horizontal-bar-chart.js

```diff
@@ -40,6 +42,60 @@ export default ChartComponent.extend(FloatingTooltipMixin,
   // Layout
   // ----------------------------------------------------------------------------
 
+  /**
+  * Overrides values in addon/mixins/axis-titles.js
+  * @override
+  */
+  xAxisPositionX: Ember.computed('graphicWidth', 'labelWidthOffset', function(){
+    return this.get('graphicWidth') / 2;
+  }),
+
+  /**
+  * X Axis Titles needs extra padding, else will intersect with the lowest bar
+  * @override
+  */
+  xAxisPositionY: Ember.computed('graphicBottom', 'axisPadding', function(){
+    return this.get('graphicBottom') + this.get('xAxisPadding');
+  }),
+
+  /**
+  * @override
+  */
+  yAxisPositionX: Ember.computed('graphicBottom', 'axisPadding', function(){
+    return -(this.get('graphicHeight') / 2);
+  }),
+
+  /**
+  * @override
+  */
+  yAxisPositionY: Ember.computed('labelWidthOffset', function(){
+    return -(this.get('labelWidthOffset'));
+  }),
+
+  /**
+  * @override
+  * MarginLeft is dependent on 'horizontalMargin'
+  *  - Otherwise, without override will result in marginLeft ~= 0
+  *  - Axis location will always be flush on the left, with labels cutoff
+  */
+  marginLeft: Ember.computed('hasAxisTitles', 'horizontalMarginLeft', 'horizontalMargin', function(){
+     return this.get('hasAxisTitles') ? this.get('horizontalMarginLeft') + this.get('horizontalMargin'): this.get('horizontalMargin');
+  }),
+
+  /**
+  * @override
+  * Anchoring using 'middle' looks better visually than using 'start'
+  */
+  updateYAxisTitle: function(){
+    this.get('yAxisTitle')
+    .text(this.get('yAxisTitleDisplayValue'))
+    .style('text-anchor', 'middle').attr({
+      x: this.get('yAxisPositionX'),
+      y: this.get('yAxisPositionY'),
+    }).attr("transform", this.get('yAxisTransform'))
+    .attr("dy", "1em");
```

Comment:
Removed, updateYAxisTitle inside axis-titles.js now uses 'middle' instead of start. This is no longer an issue and does not need to be overriden.